test/e2e/snaps/test-snap-txinsights-v2.spec.js

```diff
@@ -140,12 +127,6 @@ describe('Test Snap TxInsights-v2', function () {
           tag: 'button',
           text: 'Activity',
         });
-
-        // wait for transaction confirmation
-        await driver.waitForSelector({
```

Comment:
How come this is removed?

Reply:
Yeah, this one I can add back. Can I do it in a follow up PR?